### PR TITLE
fix softmax doc

### PR DESCRIPTION
**Pull request by hughperkins**
fairly sure these exp should be +ve, ie in http://pytorch.org/docs/master/nn.html#id29 ?

(eg see: http://pytorch.org/docs/master/nn.html#softmax or https://en.wikipedia.org/wiki/Softmax_function )

### CODE REVIEW

**Comment by hughperkins** [comment]
(Note: you can reassure yourself it should be +ve, using the following script:

```
import torch
import torch.nn.functional as F


def softmax(x):
    assert len(x.size()) == 2
    batch_size = x.size()[0]
    K = x.size()[1]
    res = torch.zeros(batch_size, K)
    exp_x = torch.exp(x)
    print('exp_x', exp_x)
    sum_exp = exp_x.sum(1)
    print('sum_exp', sum_exp)
    for n in range(batch_size):
        res[n] = exp_x[n] / sum_exp[n]
    return res


x = torch.FloatTensor([[-9, -0.3, 0.7, 1.2]])
print(softmax(x)[0])
print(F.softmax(x)[0])
```

Output:
```
exp_x 
 0.0001  0.7408  2.0138  3.3201
[torch.FloatTensor of size 1x4]

sum_exp 
 6.0748
[torch.FloatTensor of size 1]


 0.0000
 0.1219
 0.3315
 0.5465
[torch.FloatTensor of size 4]

Variable containing:
 0.0000
 0.1219
 0.3315
 0.5465
[torch.FloatTensor of size 4]
```

**Comment by soumith** [comment]
thanks!

**Comment by hughperkins** [comment]
Thanks! :)
